packages/client/README.md

```diff
@@ -466,7 +466,7 @@ The following options are available:
 
 | Logger              | Description                                      |
 | ------------------- | ------------------------------------------------ |
-| `client:fetcher`            | This option enables logging for all fetchers      |
+| `client:fetcher:#`            | This option enables logging for all fetchers      |
```

Comment:
I don't see any changes in the client code?

Reply:
I've updated the code.